worlds/noita/docs/setup_en.md

```diff
@@ -0,0 +1,41 @@
+# Noita Setup Guide
+
+## Installation
+
+### Steam
+
+Go through the standard installation process for Noita on steam.
+
+### Install Archipelago Mod
+
+Download the Archipelago mod zip from the gitHub page:
+
+[Archipelago Mod Download](https://github.com/DaftBrit/NoitaArchipelago/archive/refs/heads/master.zip)
+
+Find Noita in your steam library, right click, select Manage > Browse Local Files
+
+Here you should see your game files and a folder called "mods". Create a folder called "archipelago" and place all files
+from within the zip folder directly into the archipelago folder. After starting Noita select the Mods menu, here you 
+should see the Archipelgo mod listed.
+
+In order to enable the mod you will first need to toggle "Allow unsafe mods", this is required as some external 
+libraries are used in the mod in order to communicate with the Archipelago service. Enable allow unsafe mods and enable
+the Archipelago mod.
```

Comment:
```suggestion
In order to enable the mod, you will first need to toggle "Allow unsafe mods". This is required, as some external 
libraries are used in the mod in order to communicate with the Archipelago server. Enable "Allow unsafe mods" and enable
the Archipelago mod.
```
I don't know if the author meant "server" or not, but Archipelago isn't really a service.